retro/reco.py

```diff
@@ -175,10 +243,22 @@ def run(self, method):
                 protocol=pickle.HIGHEST_PROTOCOL
             )
 
-            dt = time.time() - t1
-            n_points = llhp.size
-            print('  ---> {:.3f} s, {:d} points ({:.3f} ms per LLH)'
-                  .format(dt, n_points, dt / n_points * 1e3))
+            if False:
+                dt = time.time() - t1
+                n_points = llhp.size
+                print('  ---> {:.3f} s, {:d} points ({:.3f} ms per LLH)'
+                      .format(dt, n_points, dt / n_points * 1e3))
+
+            #estimate = estimate_from_llhp(llhp, opt_meta)
```

Comment:
remove commented-out line unless this is useful for some reason (in which case, add comment as to why it's here)